R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -1,12 +1,13 @@
 ---
 title: "SparkR - Practical Guide"
 output:
-  html_document:
-    theme: united
+  rmarkdown::html_vignette:
     toc: true
     toc_depth: 4
-    toc_float: true
-    highlight: textmate
+vignette: >
```

Comment:
right, it seems to have a theme already and is complaining about theme already defined